app/tests/unit/pages/project-revision/[projectRevision]/index.test.tsx

```diff
@@ -325,11 +325,190 @@ describe("The Create Project page", () => {
     expect(
       screen.getByText(/test-funding-stream-description - 2020/i)
     ).toBeInTheDocument();
+    screen.logTestingPlaygroundURL();
+    // expect(screen.getByText(/test manager one/i)).toBeInTheDocument();
+    // expect(screen.getByText(/test manager two/i)).toBeInTheDocument();
+    // expect(screen.getByText(/test manager three/i)).toBeInTheDocument();
```

Comment:
Very weird that these commented out tests fail especially given what you can see from the testing playground. We should remove the logging.